server/game/gameSystems/PlayCardSystem.ts

```diff
@@ -114,7 +115,7 @@ export class PlayCardSystem<TContext extends AbilityContext = AbilityContext> ex
     }
 
     private buildPlayActionProperties(card: Card, properties: IPlayCardProperties, context: TContext, action: PlayCardAction = null) {
-        let costAdjusters = [this.makeCostAdjuster(properties.adjustCost, context)];
+        let costAdjusters = Helpers.asArray(this.makeCostAdjuster(properties.adjustCost, context));
```

Comment:
This avoid having an array containing a `null` cost adjuster if `this.makeCostAdjuster` returns `null`.